float32[19] data #numOfSensors
int16[22] currents #numOfJoints
uint8[22] temperatures #numOfJoints
uint32 timeStamp
uint32 usActuatorMode
uint32 usTimeStamp
